Add some miscellaneous useful scripts

benjaminjkraft opened the pull request:
Various things I've had sitting around.  See script docstrings for more
details.

mgersh commented:
It's useful_scripts, so if you want to just merge them and ignore this, I won't complain. That said, if you want the two that don't use script_setup to run outside of run_queries.sh, you'll need to call django.setup() at the top. And I'm curious what you meant by "needed to initialize PMOs"?

benjaminjkraft commented:
Honestly I'm not sure either is that useful outside of run_queries, but I guess it doesn't hurt to add it.

The PMOs thing is to make sure we've imported all the handler classes, so `__subclasses__` works.  Maybe we already do that somewhere?  I did it just to be safe.

benjaminjkraft commented:
I added `from script_setup import *` to both, so they can be used either way, and removed the import thing because apparently we do that already somewhere.
